docs: update  affix to data-table doc (#126)
TuSimple/naive-ui: src/alert/demos/enUS/index.demo-entry.md

| Name | Type | Default | Description |
| --- | --- | --- | --- |

> | closable | `boolean` | `false` | Can the alert information be turned off. |

Reviewer: _Alert information_ could be confused with the actual alert as a component, I would rephrase it like: _Determines whether the alert can be closed/dismissed_

> | closable | `boolean` | `false` | Can the alert information be turned off. |

Reviewer 2: ```suggestion
| closable | `boolean` | `false` | Whether the alert can be closed. |
```

| show-icon | `boolean` | `true` | Whether to show the icon of alert. |

> | title | `string` | `undefined` | Title information of alert. |

Reviewer: _Information_ is unnecessary, _Title of the alert_ should be just fine.

> | title | `string` | `undefined` | Title information of alert. |

Reviewer 2: ```suggestion
| title | `string` | `undefined` | Title of the alert. |
```

| type | `'default' \| 'info' \| 'success' \| 'warning' \| 'error'` | `'default'` | alert type. |
| on-after-leave | `Function` | `undefined` | Callback function executed when the alert disappears. |
| on-close | `() => boolean \| Promise<boolean> \| any` | `() => true` | The callback function executed when the close icon is clicked. |

## Slots

| Name    | Parameters | Description                                   |
| ------- | ---------- | --------------------------------------------- |
| default | `()`       | The content of the alert.      |

> | header  | `()`       | The content filled in the alert header.       |

Reviewer: Change _filled_ to _placed_

> | header  | `()`       | The content filled in the alert header.       |

Reviewer 2: ```suggestion
| header  | `()`       | The content placed in the alert header.       |
```

> | icon    | `()`       | The content filled in the icon part of alert. |

Reviewer: Like the comment above, and also a bit of rephrasing, since the original message is somewhat ambiguous IMO. I think a comment like: _Icon displayed in the alert_ is clearer.

> | icon    | `()`       | The content filled in the icon part of alert. |

Reviewer 2: ```suggestion
| icon    | `()`       | Icon displayed in the alert. |
```
